README.md

```diff
@@ -44,6 +44,26 @@ To run the following examples you will also need these ROS 1 packages:
 * `rostopic`
 * `rqt_image_view`
 
+### Prerequisites for the examples in this file
+
+In order to make the examples below portable between versions of ROS, we've chosen to define two environment variables, `ROS1_INSTALL_PATH` and `ROS2_INSTALL_PATH`.  These are defined as the paths to the installation location of their respective ROS versions.
+
+If you installed Noetic in the default location, then the definition of `ROS1_INSTALL_PATH` would likely be `/opt/ros/noetic`.
+
+Building the bridge as described below requires you to build all of ROS 2. I assume that you have downloaded it to `~/ros2_galactic`, and that is where you plan on building it. In this case, `ROS2_INSTALL_PATH` will be defined as `~/ros2_galactic/install`.
+
+If you've chosen to install either or both versions of ROS somewhere else, you will need adjust the definitions below to match your installation paths.
+
+Because these definitions are used continuously throughout this file, it would be useful to copy the following lines to your `.bashrc` file (if you're using `bash` as your shell).  Modify these definitions as appropriate for the versions of ROS that you're using, and for the shell that you're using.
+
+
+```bash
+export ROS1_INSTALL_PATH=/opt/ros/noetic
+export ROS2_INSTALL_PATH=~/ros2_galactic/install
```

Comment:
I'd say you have them both from binaries (building the standard bridge) or both from a workspace (building custom messages).

I cannot say what the usecase would be from sourcing ROS1 from binaries and ROS2 from source.

Reply:
Doing this is *literally* what the instructions in the same README.md says for us to do; apparently this is what is needed to build the bridge.  The only reason for these settings is because of those instructions.

Reply:
Well, in that case the comment is not directed to you but the original author :slightly_smiling_face: 

But you can ignore this then I guess.

Reply:
No problem, I understand the confusion!  When I was first going through the complete README to get the bridge working on my machine, my first thought was that I had misread things.  Turns out that they're completely accurate! :man_shrugging:

Reply:
> Turns out that they're completely accurate! man_shrugging

I didn't say that. I still think they are wrong. It's just not your doing ;-)

---
And I'm not a maintainer here, just an interested bystander.